indico/modules/rb_new/operations/rooms.py

```diff
@@ -119,9 +120,9 @@ def search_for_rooms(filters, availability=None):
         subquery = (db.session.query(RoomEquipmentAssociation)
                     .with_entities(db.func.count(RoomEquipmentAssociation.c.room_id))
                     .filter(
-                        RoomEquipmentAssociation.c.room_id == Room.id,
```

Comment:
why the reindent? i think the previous indentation was correct

(this is legacy anyway)

Reply:
It wasn't my intention to reindent this part. I'll fix it back.